slang/example/lib/i18n/strings.g.dart

```diff
@@ -1,247 +0,0 @@
-
```

Comment:
The generated file should not be deleted because it is referenced by the readme.

Reply:
Solved